pyo3-derive-backend/src/pyproto.rs

```diff
@@ -66,6 +66,7 @@ fn impl_proto_impl(
     for iimpl in impls.iter_mut() {
         if let syn::ImplItem::Method(ref mut met) = iimpl {
             // impl Py~Protocol<'p> { type = ... }
+            // TODO: MethodProto.result = meth.sig.output
```

Comment:
Not sure what this TODO is asking to happen?

Reply:
This TODO was forgotten there. It's removed.